#!/usr/bin/env python
PACKAGE = "control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("position_gain",   		 double_t, 0, "gain in the position error",         
	0.6,  0, 10)
gen.add("velocity_gain",  	 	 double_t, 0, "gain in the velocity error",         
	0.6,  0, 10)
gen.add("attitude_gain",   		 double_t, 0, "gain in the attitude error",         
	0.5,  0, 10)   #0.04
gen.add("angular_velocity_gain", double_t, 0, "gain in the angular velocity error", 
	0.1,  0, 10)   #0.01

exit(gen.generate(PACKAGE, "control", "DynamicParameters"))